README.md

```diff
@@ -6,12 +6,7 @@ Put simply: "identical", "peer" nodes that exist _in duplicate_ for the purpose
 
 # Status of Project
 
-The kamino set of tools are currently experimental and pre-alpha. There is no concrete ETA to share in expectation of an alpha or beta release, but we can share a few things about scope and priority:
-
-- Though the problem domain is general, we are focused first of all implementing solutions for Kubernetes on Azure
-- Though Kubernetes on Azure has a variety of manifestations, the initial spike will prove out functionality on [AKS Engine](https://github.com/Azure/aks-engine).
-- A working helm repo will always reflect the latest set of known-working (again: at present, only Kubernetes clusters running on Azure built with AKS Engine) functionality:
-  - https://jackfrancis.github.io/kamino/
+The kamino set of tools are currently approaching a v1.0 stable release, tested against Kubernetes running on Azure with VMSS-backed node pools, on clusters built with the [AKS Engine](https://github.com/Azure/aks-engine) tool.
```

Comment:
This all looks good.  I think we should update the walkthrough link to give both the manual and automatic mechanism walkthroughs.